docker-compose.yml

```
@@ -0,0 +1,73 @@
+      context: ./
+      dockerfile: Dockerfile
+      args:
+        - ENVIRONMENT=${ENVIRONMENT:-local}
+    volumes:
+      - ./:/app
+    ports:
+      - "8081:8081"
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER:-postgres}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-postgres}
+      - POSTGRES_HOST=${POSTGRES_HOST:-postgres}  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)